ibis/expr/tests/test_format.py

```diff
@@ -417,3 +417,24 @@ def shape(self):
 
     assert "Inc" in result
     assert last_line == "incremented: Inc(r0.a)"
+
+
+def test_format_show_variables(monkeypatch, alltypes, snapshot):
+    monkeypatch.setattr(ibis.options.repr, "show_variables", True)
+
+    filtered = alltypes[alltypes.f > 0]
+    ordered = filtered.order_by("f")
+    projected = ordered[["a", "b", "f"]]
+
+    add = projected.a + projected.b
+    sub = projected.a - projected.b
+    expr = add * sub
+
+    result = fmt(expr)
```

Comment:
What is the repr when you have an expression chained together, without intermediate variables, when `show_variables` is `True`?

Reply:
This works as it did previously it seems.